react/week5/github-search-router/github-search-router/src/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { Provider } from './context/Context';
+import {
+   BrowserRouter as Router,
+   Route,
+   Switch,
+   
+} from 'react-router-dom';
+
+import './App.css';
+
+import ListItem from './components/ListItem';
+import Form from './components/Form';
+import UserProfil from './components/UserProfil';
+
+export default class App extends Component {
+   state = {
+      usersArray: [],
+      userName: '',
+   };
+
+   render() {
+      return (
+         <>
```

Comment:
Fragments are only necessary when you want to return two or more adjacent elements. 
https://reactjs.org/docs/fragments.html